content/foundations/lifecycle.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Component lifecycle
+---
+
+These milestones summarize the maturity lifecycle of UI components in Primer. Components are required to meet all criteria _before_ being proceeding to a given milestone.
+
+## Experimental
+
+Proof-of-concept built outside of Primer.
+
+## Alpha
+
+The component is ready for preliminary usage, with breaking changes expected.
+
+- <details><summary>The component does not have external dependencies that are not relevant to end-user experiences. </summary>This could be external libraries or dependencies on libraries in an application (like GitHub.com)</details>
```

Comment:
Is there a way to not have a double negative here?

I would also show all the text at once and remove all details/summary instances from this page.

Reply:
@yaili - @broccolini wanted to use the summary/details elements to make the page less intimidating. I'm fine with going either direction though.

Reply:
I see. It makes the content much harder to read though. I was feeling lost with what was expended and what wasn't. I think if it's intimidating we have 2 options: a) accept that it is and embrace it 😆 or b) edit the content so it's less intimidating.